test/test_poolmanager.py

```diff
@@ -259,11 +259,14 @@ def test_http_connection_from_context_case_insensitive(self) -> None:
         assert pool is other_pool
         assert all(isinstance(key, PoolKey) for key in p.pools.keys())
 
-    def test_deprecated_no_scheme(self) -> None:
+    @patch("urllib3.poolmanager.PoolManager.connection_from_host")
+    def test_deprecated_no_scheme(self, connection_from_host: mock.MagicMock) -> None:
+        connection_from_host.side_effect = ConnectionError("Not attempting connection")
```

Comment:
Please add a comment describing that this side effect exists only to allow the test to pass without internet access